signing_clients/apps.py

```diff
@@ -102,6 +106,21 @@ def _digest(data):
     return {'md5': md5.digest(), 'sha1': sha1.digest()}
 
 
+def get_addon_type_from_signature(pkcs7):
+    info = SignatureInfo(pkcs7)
+    ou = get_signer_organizational_unit_name(info)
+
+    if ou == constants.OU_PRODUCTION:
+        # TODO: where should the CN be taken from to identify
+        # hotfix-addons? `if endEntity.CN = "firefox-hotfix@mozilla.org"`
```

Comment:
@jvehent based on https://github.com/mozilla/addons-server/issues/5619#issuecomment-309033995 - what's `endEntity` in your code snippet referencing to in a real signature?